lib/elixir/lib/kernel/utils.ex

```diff
@@ -110,12 +112,15 @@ defmodule Kernel.Utils do
   def raise(msg) when is_binary(msg) do
     RuntimeError.exception(msg)
   end
+
```

Comment:
Let's not add those in this commit as you are not even changing those lines and `destructure` above does not introduce new lines either.